chainweb.cabal

```diff
@@ -437,7 +437,7 @@ test-suite chainweb-tests
     default-language: Haskell2010
     ghc-options:
         -threaded
-        -with-rtsopts=-N2
+        -with-rtsopts=-N4
```

Comment:
rationale?

Reply:
This PR takes a lot of sequential test groups and makes them parallel, so we have more opportunities for parallelism between test suites. And I don't really know why it was -N2, it was a bit faster for me using -N4.